hrm-service/src/permissions/middlewares.js

```diff
@@ -27,6 +28,24 @@ const canEditCase = asyncHandler(async (req, res, next) => {
   next();
 });
 
+const canEditContact = asyncHandler(async (req, res, next) => {
+  if (!req.isAuthorized()) {
+    const { accountSid, user, can } = req;
+    const { contactId } = req.params;
+
+    const contactObj = await getContactById(accountSid, contactId);
+    const contactModel = new Contact(contactObj);
```

Comment:
We can remove this and the same workaround in the case middleware once cancan goes